platform/android/java/lib/src/org/godotengine/godot/input/GodotEditText.java

```diff
@@ -161,13 +164,25 @@ public boolean onKeyDown(final int keyCode, final KeyEvent keyEvent) {
 	// ===========================================================
 	// Methods
 	// ===========================================================
-	public void showKeyboard(String p_existing_text, int p_max_input_length) {
-		mOriginText = p_existing_text;
+	public void showKeyboard(String p_existing_text, int p_max_input_length, int p_cursor_start, int p_cursor_end) {
+
```

Comment:
Remove the empty line at the start of the block (`clang-format` should now do it automatically).